### PR TITLE
Explain as human readable datetime instead of epoch

### PR DESCRIPTION
**Pull request by doheonlee**
Ninja explains output is older than most recent input with their epoch.
Datetime is changed from epoch value to datetime human reads easily.

### CODE REVIEW

File: src/graph.cc [hunk 1]
```diff
@@ -174,11 +174,22 @@ bool DependencyScan::RecomputeOutputDirty(Edge* edge,
     }
 
     if (output_mtime < most_recent_input->mtime()) {
+      time_t output_t = static_cast<time_t>(output_mtime);
+      time_t most_recent_input_t = static_cast<time_t>(most_recent_input->mtime());
```
Comment (evmar) [inline src/graph.cc]: On Windows, this value is not a time_t.  I think we need a function that converts these mtimes to human-readable times that is implemented differently on Linux and Windows.

Comment (evmar) [inline src/graph.cc]: (It'd be fine to just report the number as-is on Windows. It's just that passing it to ctime isn't right.)

**Comment by doheonlee** [comment]
Dear @evmar , thank you very much for your detailed reviewing action :)
I uploaded new commit for some modification.

First, let this patch works on not WINDOWS platform and wrapped it with g_explaning condition.
EXPLAIN macro acts only g_explaining is true, thus this wrapping does not need else condition.

**Comment by jhasse** [comment]
Sorry that this has been open for such a long time without a comment.

I think that having a consistent message on all platforms is more important though. Maybe we can revisit this later, closing for now.

**Pull request closed without merging**